src/app.js

```diff
@@ -0,0 +1,91 @@
+let tasks = [];
+
+const setItems = () => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+const getItems = () => {
+  const data = JSON.parse(localStorage.getItem('tasks'));
+  if (data) {
+    tasks = data;
+  }
+};
+
+export const renderTasks = () => {
+  const taskList = document.querySelector('.input');
+  taskList.innerHTML = '';
+  getItems();
+
+  tasks.forEach((task) => {
+    const listItem = document.createElement('li');
+    const checkbox = document.createElement('input');
+    const inputText = document.createElement('input');
+    const menuImg = document.createElement('div');
+    inputText.className = 'text';
+    inputText.type = 'text';
+    inputText.value = task.description;
+    inputText.addEventListener('blur', (e) => {
+      task.description = e.target.value;
+      setItems();
+    });
+    inputText.style.width = '20rem';
+    menuImg.className = 'menu';
+    menuImg.innerHTML = '<i class="fa fa-ellipsis-v" aria-hidden="true"></i>';
+    menuImg.addEventListener('click', (e) => {
+      e.preventDefault();
+      const section = document.createElement('div');
+      const change = document.createElement('button');
+      change.className = 'white';
+      const clickImg = document.querySelector('.fa-ellipsis-v');
```

Comment:
Very nice effort implementing the icon to delete the item. But when that icon is clicked, a recycle bin icon appears along with this icon, and it will keep generating the icons when we keep clicking this icon.
So hide this Icons when recycle bin icon is displayed.
![image](https://user-images.githubusercontent.com/61361037/229117020-7a7dd779-2120-4f0a-9fad-15710f7ab38c.png)
So, when clicked again and again, it gives:
![image](https://user-images.githubusercontent.com/61361037/229117152-361e7234-4acc-4cfd-b3d4-5020ea7b3220.png)

2. It don't delete the item, if it is not on the top of the list.
![image](https://user-images.githubusercontent.com/61361037/229119295-284c83f9-332c-453f-acd5-cf3c7ea11185.png)

Kindly fix these issues.